src/components/LevelBadge.tsx

```diff
@@ -0,0 +1,54 @@
+import { RegexContext } from "../store/regex-context"
+import { useContext } from "react"
+import classes from "./LevelBadge.module.css"
+import Newbee from '../assets/newbee.png'
+import Pro from '../assets/pro.png'
+import Mid from '../assets/mid.png'
+import {millisecondsToMinutes} from 'date-fns'
+
+type TimeProps = {
+    time: number
+}
+
+const LevelBadge: React.FC<TimeProps> = (props) => {
+    const regExContext = useContext(RegexContext)
+    const {numberOfGuessedWords} = regExContext
+    let convertedTimeResult = millisecondsToMinutes(props.time)
+
+    let level;
```

Comment:
it isn't a good practice to write if statements in this way

Reply:
Maybe I can transform these if-s into a switch statement? Or is it also a bad practice?

Reply:
Found the optimized way!
    let level: string = 'NEWBEE'
   numberOfGuessedWords > 15 && numberOfGuessedWords < 25 ? level = 'MID' : level = 'PRO'